llvm/lib/Support/RISCVISAInfo.cpp

```diff
@@ -695,6 +695,106 @@ RISCVISAInfo::parseNormalizedArchString(StringRef Arch) {
   return std::move(ISAInfo);
 }
 
+static Error splitExtsByUnderscore(StringRef Exts,
+                                   std::vector<std::string> &SplitedExts) {
```

Comment:
`SplitExts`

Splitted has two `t`s but I don't think the `ed` is needed.

Reply:
Done